mast/templates/user_settings.html

```diff
@@ -1,3 +1,76 @@
 {% extends "main_layout_template.html" %}
```

Comment:
Same as above. Please provide some kind of validation defined in forms.py on the frontend. Fill free to refer to: https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation